sgqlc/types/__init__.py

```diff
@@ -2176,6 +2178,8 @@ def _to_graphql_name(cls, name):
         >>> BaseItem._to_graphql_name('__typename__')
         '__typename'
         '''
+        if not CONVERT_PYNAMES:
+            return name
```

Comment:
would need to move this to the schema, document and add a doctest to cover that.

But as said, I don't see why this would be needed. Notice that you can describe the GraphQL name when declaring fields or arguments, the code-generator already does that.